src/node/src/client.js

```diff
@@ -224,9 +225,11 @@ function _read(size) {
     } catch (e) {
       self._readsDone({code: grpc.status.INTERNAL,
                        details: 'Failed to parse server response'});
+      return;
```

Comment:
returning here finishes to emit `error`/`status` and avoid events in future reads?